### PR TITLE
Update devcontainer.json to include swift

### LINKED ISSUE #7652: Swift missing from .devcontainer
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

swift

### Package manager version

_No response_

### Language version

_No response_

### Manifest location and content before the Dependabot update

_No response_

### dependabot.yml content

_No response_

### Updated dependency

_No response_

### What you expected to see, versus what you actually saw

Expected the devcontainer to contain swift entries.

What I saw was it did not exist.

### Native package manager behavior

_No response_

### Images of the diff or a link to the PR, issue, or logs

_No response_

### Smallest manifest that reproduces the issue

_No response_

### PR DESCRIPTION
**Pull request by dwc0011**
Fixes #7652 by adding swift to core-dev/devcontainer.json

### CODE REVIEW

**Review by Nishnha — APPROVED** [review]

File: .devcontainer/core-dev/devcontainer.json [hunk 1]
```diff
@@ -82,6 +82,8 @@
     "-v", "${localWorkspaceFolder}/python/Gemfile:/home/dependabot/dependabot-core/python/Gemfile",
     "-v", "${localWorkspaceFolder}/python/lib:/home/dependabot/dependabot-core/python/lib",
     "-v", "${localWorkspaceFolder}/python/spec:/home/dependabot/dependabot-core/python/spec",
+    "-v", "${localWorkspaceFolder}/swift/lib:/home/dependabot/dependabot-core/swift/lib",
+    "-v", "${localWorkspaceFolder}/swift/spec:/home/dependabot/dependabot-core/swift/spec",
```
Comment (deivid-rodriguez) [inline .devcontainer/core-dev/devcontainer.json:86]: I see that other ecosystems also add the `.gemspec` and `Gemfile` files, should we add them here too, or remove the others if not needed?

Comment (abdulapopoola) [reply .devcontainer/core-dev/devcontainer.json:86]: I think @Nishnha published a Gem for Swift; should we add it for consistency?

Comment (dwc0011) [reply .devcontainer/core-dev/devcontainer.json:88]: Done

Comment (Nishnha) [reply .devcontainer/core-dev/devcontainer.json:88]: yeah we should also mount those

**Review by Nishnha — APPROVED** [review]

**Pull request merged**